Analysis/HiggsTauTau/src/HTTRun2RecoilCorrector.cc

```diff
@@ -151,9 +155,21 @@ namespace ic {
       corrector_->Correct(mvaMetx,mvaMety,genpX,genpY,vispX,vispY,njets,correctedMetx,correctedMety); 
     }
     //Now stick this back into our met object:
+    //if(met_label_ == "pfMET"){
+    //   std::cout<< "Before Recoil corrections:" << std::endl;
+    //   std::cout << "MET Pt = " << mvaMet->pt() << std::endl;
+    //   std::cout << "MET phi = " << mvaMet->phi() << std::endl;
+    //   std::cout << "MET Energy = " << mvaMet->energy() << std::endl;
+    //}
     mvaMet->set_pt(sqrt(correctedMetx*correctedMetx+correctedMety*correctedMety));
     mvaMet->set_phi(atan2(correctedMety,correctedMetx));
     mvaMet->set_energy(sqrt(met_res_e*met_res_e+correctedMetx*correctedMetx+correctedMety*correctedMety));
```

Comment:
The fact that the met is labelled mvaMet is of course entirely my fault, and it doesn't really matter as the correct version of met is extracted at the beginning and so the label doesn't matter, but at some point in the future it might be good to relabel it.